src/MassTransit/DiagnosticSourceExtensions.cs

```diff
@@ -25,9 +25,8 @@ public static void AddSendContextHeaders(this StartedActivity? startedActivity,
             activity.AddTag(DiagnosticHeaders.SourceAddress, context.SourceAddress);
             activity.AddTag(DiagnosticHeaders.DestinationAddress, context.DestinationAddress);
 
-            activity.AddTag(DiagnosticHeaders.Kind, DiagnosticHeaders.SpanKind.Producer);
-            activity.AddTag(DiagnosticHeaders.PeerHost, context.DestinationAddress.Host);
-            activity.AddTag(DiagnosticHeaders.PeerAddress, context.DestinationAddress.AbsolutePath);
+            activity.AddTag(DiagnosticHeaders.ServiceKind, DiagnosticHeaders.Kind.Producer);
+            activity.AddTag(DiagnosticHeaders.DestinationHost, context.DestinationAddress.Host);
```

Comment:
This is worthless, it's just the message broker host name.

Reply:
I don't have a particular use for it - really just trying to make use of peer.host. I guess i could see some scenarios where someone may be working with different message broker instances and wants to isolate logs in that way.